schemas/sublime-base.json

```diff
@@ -15,6 +15,39 @@
     },
     "VSCodeSchemaExtensionProperties": {
       "properties": {
+        "defaultSnippets": {
```

Comment:
In what way is defaultSnippets used?

Reply:
There is one example here: https://github.com/sublimelsp/LSP-json/blob/205853c7085dc86c79e8b7f5e4c32a2995b8157f/sublime-package.json#L26-L34

It allows registering extra completions (suggestions) for that object. Like `default` and `examples` already do but with more control (`examples` can just be snippets and both can't have a custom label or documentation text)